index.html 

```diff
@@ -3,25 +3,33 @@
 	<title>Bubble Tea site</title>
 </head>
 <body>
-	<div class="topnav">
-		<img src="" alt="logo" style="width:600px; height:400px"/>
-		<a class="active" href=#home">Home</a>
-		<a href=#news">News</a>
-		<a href=#about">About</a>
-		<a href=#contact”>Contact</a>
-	</div
-	<div>
-		<img src="" alt="description of image">
-	</div>
+	<nav>
+		<div class="topnav">
+			<img src="" alt="logo" style="width:600px; height:400px"/>
+			<a class="active" href=#home">Home</a>
```

Comment:
Wrap the below anchor tags in ul element, and for each anchor element, wrap that in li element
Example:
```
<ul>
  <li><a>Home</a></li>
  <li><a>News</a></li>
<ul>
```

Reply:
do the above for the anchor tags in `<footer>` below as well